src/java/com/LoginServlet.java

```diff
@@ -73,15 +63,52 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response)
     protected void doPost(HttpServletRequest request, HttpServletResponse response)
             throws ServletException, IOException {
         processRequest(request, response);
+
+        String message = null;
+
+        UserManagement loginUser = new UserManagement();
+        loginUser.setUsername(request.getParameter("username"));
+        loginUser.setPassword(request.getParameter("password"));
+
+        //String user = AlphacabListener.login(loginUser);
+        UserManagement userType = new UserManagement();
+        loggedInUser.getUserType;// Not sure how to do this.
+
+        if (loginUser.equals(null)) {
+
+            message = "Please try again";
+
+        } else {
+
```

Comment:
The `message` assignment is the same no matter the user type here.  So you could just set it once on line 82, rather than setting it over again in each condition statement.